### PR TITLE
fix(raml): update create account schema to work with multiple forms

### PR DESCRIPTION
**Pull request by abuecker**
Since the multiple forms have colliding names or different formatted field names, we need a way to isolate the values.

### CODE REVIEW

**Comment by abuecker** [comment]
@zrangerz @olmstead @johntowell please review.

**Comment by zrangerz** [comment]
I see that you are managing for non globally unique field names so need to map say field name: text1 in ACH so that it doesn’t collide with field name: text1 in taxable.

+1

--
Steve Mays
Sent with Airmail

**Comment by johntowell** [comment]
We have the following template constants on the server
```
    ACCOUNT_APPLICATION_FORM,
    MOVE_MONEY_FORM,
    IRA_FORM,
    IRA_CUSTODIAL_ACCOUNT_AGREEMENT,
    IRA_MOVE_MONEY,
    ACCOUNT_TRANSFER_FORM,
    ELECTRONIC_FUNDS_TRANSFER
```
Could we use those somehow? Also could we come up with a format that is flexible to allow new/different templates?

**Comment by abuecker** [comment]
Sure, I'll use that format.  Going to need one more for `IRA_DEPOSIT_SLIP`.

**Comment by johntowell** [comment]
An array of maps?

```
docusignFormData: [
{
   template: ACCOUNT_APPLICATION_FORM,
   formData: [
     {
      "formType": "TEXT",
       "label": "FirmName",
       "value": "Trizic"
     },
     {
       "formType": "RADIO",
       "label": "AccountType",
       "value": "Individual"
     },
     {
       "formType": "SELECT",
       "label": "PhysicalAddressState",
       "value": "CA"
     }
   ]
},
{
   template: MOVE_MONEY_FORM,
   formData: [
     {
      "formType": "TEXT",
       "label": "FirmName",
       "value": "Trizic"
     },
     {
       "formType": "RADIO",
       "label": "AccountType",
       "value": "Individual"
     },
     {
       "formType": "SELECT",
       "label": "PhysicalAddressState",
       "value": "CA"
     }
   ]
}
]
```

**Comment by abuecker** [comment]
Cool.  I like that format.

**Comment by abuecker** [comment]
@johntowell made the changes.

File: api/v3/accounts/examples/accounts.req.json [hunk 1]
```diff
@@ -3,24 +3,66 @@
   "fundingType": "Cash",
   "portfolioId": 2462,
   "questionnaire":"questionnaire string",
-  "name": "Bob Sample",
-  "jointName": "Jane Sample",
-  "jointEmail": "jane.sample@example.com",
```
Comment (johntowell) [inline api/v3/accounts/examples/accounts.req.json]: you need to add the example back in right?

**Comment by johntowell** [comment]
👍

**Pull request closed without merging**